Source/Core/DolphinQt/Config/VerifyWidget.cpp

```diff
@@ -74,11 +74,17 @@ void VerifyWidget::CreateWidgets()
   std::tie(m_md5_checkbox, m_md5_line_edit) = AddHashLine(m_hash_layout, tr("MD5:"));
   std::tie(m_sha1_checkbox, m_sha1_line_edit) = AddHashLine(m_hash_layout, tr("SHA-1:"));
 
+  const auto default_to_calculate = DiscIO::VolumeVerifier::GetDefaultHashesToCalculate();
+  m_crc32_checkbox->setChecked(default_to_calculate.crc32);
+  m_md5_checkbox->setChecked(default_to_calculate.md5);
+  m_sha1_checkbox->setChecked(default_to_calculate.sha1);
+
   m_redump_layout = new QFormLayout;
   if (DiscIO::IsDisc(m_volume->GetVolumeType()))
   {
     std::tie(m_redump_checkbox, m_redump_line_edit) =
         AddHashLine(m_redump_layout, tr("Redump.org Status:"));
+    m_redump_checkbox->setChecked(CanVerifyRedump());
```

Comment:
If you need this line of code, you also need to call `UpdateRedumpEnabled`. But because of the change I asked you to make... I don't think this line is actually necessary anymore?

Reply:
It (or something like it ) is needed because AddHashLine used to do `checkbox->setChecked(true);` for `m_redump_checkbox` but now it does not. I don't think UpdateRedumpEnabled is required because the item was just created and therefor is in enabled state.

Reply:
Ah, yeah, you do need something that sets it to true. But what I mean is, if `CanVerifyRedump` were to return false here, you would need to set enabled to false. You aren't checking if you should set enabled to false, yet you are checking if you should set checked to true. The code works, but it's confusing to read. You should either have a hardcoded true when calling setChecked or use the result of `CanVerifyRedump` for setting both checked and enabled.